js/index.js

```diff
@@ -1 +1,138 @@
 // Your code goes here
+
+const nav = document.querySelector('.nav');
+const logo = document.querySelector('.logo-heading');
+const bus = document.querySelector('header img');
+const img1 = document.querySelector('.img-content:nth-of-type(1)');
+const img2 = document.querySelector('.img-content:nth-of-type(2)');
+const h2 = document.querySelector('h2');
+const boatImg = document.querySelector('.content-destination img');
+const div1 = document.querySelector('.destination:nth-of-type(1)');
+const div2 = document.querySelector('.destination:nth-of-type(2)');
+const div3 = document.querySelector('.destination:nth-of-type(3)');
+const section = document.querySelector('.content-pick');
+const a1 = document.querySelector('nav a:nth-of-type(1)');
+const a2 = document.querySelector('nav a:nth-of-type(2)');
+const a3 = document.querySelector('nav a:nth-of-type(3)');
+const a4 = document.querySelector('nav a:nth-of-type(4)');
+
+// 1 - mouseover
+nav.addEventListener('mouseover', e => {
+    e.target.style.fontSize = '5rem';
+    e.target.style.color = 'red';
+})
+
+// 2 - mouseout
+nav.addEventListener('mouseout', e => {
+    e.target.style.fontSize = '1.75rem';
+    e.target.style.color = 'black';
+})
+
+// 3 - click
+logo.addEventListener('click', e => {
+    if(e.target.style.color === 'white') {
+        e.target.style.color = 'black';
+}   else {
+    e.target.style.color = 'white';
+}
+})
+
+// 4 - mouseenter
+bus.addEventListener('mouseenter', e => {
+    e.target.style.borderRadius = '50%';
+})
+
+// 5 - mouseleave
+bus.addEventListener('mouseleave', e => {
+    e.target.style.borderRadius = '0';
+})
+
+// 6 - wheel
+img1.addEventListener('wheel', e => {
+    e.target.style.transform = 'rotate('+window.pageYOffset+'deg)';
+})
+
+img2.addEventListener('wheel', e => {
+    e.target.style.transform = 'rotate('+window.pageYOffset+'deg)';
+})
+
+// 7 - drag/drop
```

Comment:
Very nice job on this sir, especially getting this drag and drop working. I love this effect and how you got it to work here. I  feel I am slightly weak in making this work correctly still. So I am excited you got it working so quickly in your journey!  A+ This I must say is the highlight of this page.